linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/rule.yml

```diff
@@ -12,6 +12,8 @@ description: |-
     <pre>action(type="omfwd" protocol="tcp" Target="&lt;remote system>" port="6514"
         StreamDriver="gtls" StreamDriverMode="1" StreamDriverAuthMode="x509/name")
     </pre>
+    Replace the <tt>&lt;remote system></tt> in the above configuration with an IP address or a host name of the remote logging server.
```

Comment:
Could you also add the command to achieve compliance?
A command that echoes the settings into the config file, probably above this line. As the "Replace the <tt>&lt;remote system></tt>  " text goes well with it.